torch/csrc/jit/codegen/cuda/README.md

```diff
@@ -224,7 +224,7 @@ There're a few debug dump that could be turned on via environment variables. Loo
 
 1. There's regression after turning on nvfuser.
 
-First thing is to check that you have fusion kernel running properly. Try to run your model with fallback disabled to see if you hit any errors that caused fallback via `export PYTORCH_NVFUSER_DISABLE_FALLBACK=1`.
+First thing is to check that you have fusion kernel running properly. Try to run your model with fallback disabled to see if you hit any errors that caused fallback via `export PYTORCH_NVFUSER_DISABLE=fallback`.
```

Comment:
nitpick: Can we have a quick note on debugging/tuning knobs? Maybe only listing the important ones (e.g. fma, fallback) and point people to the code. Either have that in FAQs, or add a section following `Query nvfuser codegen kernels`

Reply:
I'll add some short note to the README file.

Reply:
Thanks looks good enough~